texmf/tex/latex/spherex/spherex.cls

```diff
@@ -197,7 +207,7 @@
 \topmargin = -0.5in
 \headwidth = 6.5in
 \lhead{\includegraphics[height=0.5in]{logo.pdf}}
-\chead{\large\ \@docCategoryName \vspace{10pt}\\\@docShortTitle}
+\chead{\large\ \@docSubsystem\ \@docCategoryName \vspace{8pt}\\\@docShortTitle}
```

Comment:
The "8pt" thing here is a total kludge, not to mention a wild guess.  The issue is that the line spacing in the `\chead` is different from the `\rhead` - the top line in the middle has a higher baseline than the top line on the right.  I am pretty sure that this is because `\large` brings with it a bigger `\baselineskip` than the default font size.  The old `10pt` was just getting added to each.

After sleeping on it, some of my ancient TeX knowledge came back and I wonder if this problem would be better solved with a couple of `\vphantom`s to equalize the spacing.

Reply:
It looks like the `8pt` was very close to exactly what's needed.  It's a kludge but I'm OK with that for now.